Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -179,6 +179,20 @@ extension AlertManager {
                 }
             }
         }
+        alertStore.lookupAllAcknowledgedUnretractedRepeatingAlerts {
+            switch $0 {
+            case .failure(let error):
+                self.log.error("Could not fetch acknowledged unretracted repeating alerts: %@", error.localizedDescription)
+            case .success(let alerts):
+                alerts.forEach { alert in
+                    do {
+                        self.replayAlert(try Alert(from: alert, adjustedForStorageTime: true))
+                    } catch {
+                        self.log.error("Error decoding alert from persistent storage: %@", error.localizedDescription)
+                    }
+                }
+            }
+        }
```

Comment:
@rickpasetto While this works for this use case, I'm curious if there are other recurring alerts that should not be rescheduled on launch after they are acknowledged?

Reply:
I don't think you need this.  If it is acknowledged, it shouldn't play back.  I need to understand your use case more.